larky/src/main/resources/vendor/Crypto/OpenPGP/__init__.star

```diff
@@ -0,0 +1,265 @@
+load("@stdlib//larky", WHILE_LOOP_EMULATION_ITERATION="WHILE_LOOP_EMULATION_ITERATION", larky="larky")
+load("@stdlib//struct", struct="struct")
+load("@vendor//Crypto/Util/py3compat", tobytes="tobytes", bord="bord", tostr="tostr")
+load("@vendor//option/result", Error="Error")
+
+StopIterating = Error("StopIteration")
```

Comment:
This is now StopIteration. You can see the test_builtins.star on how to use it. https://github.com/verygoodsecurity/starlarky/blob/e72ace569282a7cd6dd36aace70fb9178e499cda/larky/src/test/resources/stdlib_tests/test_builtins.star#L63

Reply:
got it, thanks!